### PR TITLE
fix CI

### PR DESCRIPTION
**Pull request by guybedford**
This should fix the Wasmtime install, which I believe is due to the Rust toolchain lock to version `1.68.3`. In addition adds Wizer and wasm-tools installs.

### CODE REVIEW

**Comment by tschneidereit** [comment]
These tools are all installed by the cmake script as well, so it'd be great if we could use those versions instead of building them from source for every run.

Perhaps a good way to enable this would be to introduce a CMake target that prints a `PATH` with the relevant directories added, and then have CI use that to modify the `PATH`?

**Comment by guybedford** [comment]
Okay done, it seems like GitHub actions doesn't like altering the PATH for some reason, so I've used explicit WIZER and WASM_TOOLS env vars in the componentize script.

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -22,6 +22,9 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-latest]
+        wasmtime: [19.0.2]
+        wizer: [5.0.0]
+        wasm-tools: [1.203.0]
```
Comment (tschneidereit) [inline .github/workflows/main.yml:27]: Is this change still needed?

**Review by tschneidereit — APPROVED** [review]
This is great, thank you!

File: .github/workflows/main.yml (hunk 1 above)
Comment (guybedford) [reply .github/workflows/main.yml]: Good catch, thanks.
